problems/2.functions.js

```diff
@@ -2,16 +2,21 @@
 * @param {string} name - The name of the user to greet
 * @returns {null}
 */
-function logGreeting() {
-
+function logGreeting(name) {
```

Comment:
This function should not have conditions. It's asking you to console.log. No return .